packages/xrpl/test/setupClient.ts

```diff
@@ -1,64 +1,96 @@
-/* eslint-disable no-param-reassign -- Necessary for test setup */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types -- Necessary for test setup */
-import { Client, BroadcastClient } from 'xrpl-local'
+import { Client } from '../src/client'
+// eslint-disable-next-line import/no-deprecated -- Will remove in 3.0.0
+import BroadcastClient from '../src/client/BroadcastClient'
 
-import createMockRippled from './createMockRippled'
-import { getFreePort } from './testUtils'
+import createMockRippled, {
+  type MockedWebSocketServer,
+} from './createMockRippled'
+import { destroyServer, getFreePort } from './testUtils'
+
+export interface XrplTestContext {
```

Comment:
I'd prefer if this was two different interfaces for clients vs broadcast clients (to simplify types and make it so you don't need to use the `!` with the `testContext.mockRippled`), but since the broadcast client tests will be removed in a followup PR I don't think it's worth doing

Reply:
The types can get refactored then I'd say

Reply:
Yeah - that can be done in a followup